src/main/kotlin/team/themoment/gsmNetworking/domain/user/service/ProfileUrlRegistrationService.kt

```diff
@@ -0,0 +1,37 @@
+package team.themoment.gsmNetworking.domain.user.service
+
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import team.themoment.gsmNetworking.common.exception.ExpectedException
+import team.themoment.gsmNetworking.domain.user.domain.User
+import team.themoment.gsmNetworking.domain.user.repository.UserRepository
+
+@Service
+@Transactional(rollbackFor = [Exception::class])
+class ProfileUrlRegistrationService(
+    private val userRepository: UserRepository
+) {
+
+    fun execute(authenticationId: Long, profileUrl: String) {
+        val user = userRepository.findByAuthenticationId(authenticationId)
+            ?: throw ExpectedException("user를 찾을 수 없습니다.", HttpStatus.NOT_FOUND)
+
+        profileUrlRegistered(user, profileUrl)
+    }
+
+    fun profileUrlRegistered(user: User, profileUrl: String) {
+        val addProfileUrlToUser = User(
```

Comment:
`addProfileUrlToUser`는 변수에서 사용되기에 적절하지 않은 이름이라고 생각합니다.   
오히려 함수에 더 적절한 이름 같아요.

우측 행동의 결과를 이후 부여받는 변수이기 때문에 `userUpdatedProfileUrl`나 `userAddedProfileUrl` 가 더 적절합니다.


```suggestion
        val userUpdatedProfileUrl = User(
```

Reply:
수정하였습니다.
[9753c1d](https://github.com/themoment-team/GSM-Networking-server/pull/73/commits/9753c1d3d6b4158a8293272a8782e5494556633d)